pkg/Microsoft.NETFramework.Compatibility.Shims/Microsoft.NETFramework.Compatibility.Shims.pkgproj

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <PackageVersion>$(CompatibilityShimsPackageVersion)</PackageVersion>
+    <_ShimsLocationPath>$(BaseIntermediateOutputPath)shims/netcoreapp/facades/</_ShimsLocationPath>
+    <_PkgLibDir>/lib/netcoreapp2.0/</_PkgLibDir>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ShimsToInclude Include="mscorlib" />
```

Comment:
How did you come up with this list?

Reply:
He found the set that points to anything in the compat meta-package.

@safern Did you find the list of dangling type-forwards from this set?